apps/web-giddh/src/app/invoice/invoice.module.ts

```diff
@@ -170,13 +168,16 @@ import { GenerateEWayBillModule } from './preview/models/generateEWayBill/genera
         MatTabsModule,
         BulkExportVoucherModule,
         FroalaTemplateEditorModule,
-        GenerateEWayBillModule
+        GenerateEWayBillModule,
+        InvoiceSettingModule,
+        InvoiceGenerateModule
     ],
     exports: [
         InvoiceRoutingModule,
         TooltipModule,
         DownloadOrSendInvoiceOnMailComponent,
-        InvoicePreviewComponent
+        InvoicePreviewComponent,
+        EditInvoiceComponent
```

Comment:
declare this in invoice template module and export there.

Reply:
Done